### PR TITLE
Jira channel

### PR DESCRIPTION
**Pull request by bamandel**
This has MOST of the functionality we wanted for the Jira channel. There are a few cases where we don't have the information we want to and a fixme should be there explaining what we want or need.

There are also Freemarker template service changes here which I originally planned on using but ended up not.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -80,7 +80,7 @@ dependencies {
     implementation project(':alert-database')
 
     implementation 'com.synopsys.integration:polaris-common:0.12.0'
-    implementation 'com.synopsys.integration:int-jira-common:0.0.2'
```
Comment (gkillough) [inline build.gradle]: 0.0.4 is now available

File: src/main/java/com/synopsys/integration/alert/channel/ChannelFreemarkerTemplatingService.java [hunk 2]
```diff
@@ -30,39 +30,40 @@
...
-    private final Configuration configuration;
 
-    public ChannelFreemarkerTemplatingService(final String templateDirectory) throws IOException {
-        this.templateDirectory = templateDirectory;
-        this.configuration = createFreemarkerConfig();
+    public String getTemplatePath(final String templatesDirectory, final String channelDirectory) {
+        if (StringUtils.isNotBlank(templatesDirectory)) {
+            return String.format("%s/%s", templatesDirectory, channelDirectory);
+        }
+        return String.format("%s/src/main/resources/%s/templates", System.getProperties().getProperty("user.dir"), channelDirectory);
```
Comment (gkillough) [inline src/main/java/com/synopsys/integration/alert/channel/ChannelFreemarkerTemplatingService.java]: For production the templates are exported to a different location to be accessible in the docker container. See _distributions_ in the build.gradle.

Comment (psantos1113) [reply src/main/java/com/synopsys/integration/alert/channel/ChannelFreemarkerTemplatingService.java]: We should refactor this to use AlertProperties.getAlertTemplatesDir() at this point.  This is already used to get the templates. We should also refactor the build.gradle to pass the property in via the command line.  Also update the application properties file used for the tests to set this alert.templates.dir as well.  This way we can remove line 50.

Comment (bamandel) [reply src/main/java/com/synopsys/integration/alert/channel/ChannelFreemarkerTemplatingService.java]: When you say update build.gradle, will I only be updating the runServer command to add this property value? I'm not aware of anywhere else I'd need to make changes for this.

**Pull request merged**